src/index.js

```diff
@@ -1,3 +1,9 @@
 import auditor from "./auditor.js";
 
-auditor();
+const URL = "https://accesslint-service-staging.herokuapp.com/api/v1/reports";
+
+window.AccessLint = {
+  audit({ url = URL, raiseError = false }) {
```

Comment:
Expected ',' and instead saw '='.<br>Expected an identifier and instead saw 'false' (a reserved word).<br>'false' is defined but never used.